generation/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -18,7 +18,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        os.makedirs(root_dir)

Should have the <code>MONAI_DATA_DIRECTORY</code> logic here, since it is mentioned in the comments.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1953/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Hi @Can-Zhao, could you please help address this, then I will merge the pr. Thanks.

Reply:
I will remove MONAI_DATA_DIRECTORY logic to make it consistent with the other two training notebook

Reply:
Done! Thanks